fiftyone/utils/annotations.py

```diff
@@ -978,7 +979,7 @@ def _format_attributes(backend, attributes):
                 )
             )
 
-        _attr = {"type": attr_type}
+        _attr["type"] = attr_type
```

Comment:
Not a 100% sure how this works, we already are getting `attr_type` from `_attr` in line 959 and `attrr_type` is not modified or updated in between?

Reply:
If @ehofesmann implements my suggestion above, then this line would be required.

As currently implemented, I agree it is a no-op.